roles/wordpress-setup/defaults/main.yml

```diff
@@ -0,0 +1 @@
+site_uses_local_db: "{{ site_env.db_host == 'localhost' }}"
```

Comment:
We could. I kind of thought about that but figured there's no reason for anyone to put `127.0.0.1`? Trellis configures localhost automatically.

Want to do a PR?